apis/filesystem/src/main/java/org/jclouds/filesystem/util/Utils.java

```diff
@@ -49,7 +55,23 @@ public static void deleteRecursively(File file) throws IOException {
             }
          }
       }
-      Files.delete(file.toPath());
+
```

Comment:
I should have explained further. Yes, this PR is mostly to make the tests compatible with virtually any windows virus checker.